turbinia/task_manager.py

```diff
@@ -135,6 +137,30 @@ def _backend_setup(self, *args, **kwargs):
     """
     raise NotImplementedError
 
+  @staticmethod
+  def validate_request_recipe(request_recipe):
```

Comment:
See my comments below, but we can probably remove this method if we refactor out the 'task_recipes' layer and just use the `recipe_helpers.validate_recipe` method directly.